### PR TITLE
Pd 473 create task snapshot article content

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates the CLISnapshotTask.md article by adding the common CLI namespace content and the commands for this namespace.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALECLIReference/Task/CLISnapshotTask.md [hunk 1]
```diff
@@ -6,12 +6,370 @@ aliases:
...
+{{< expand "Command Example" "v" >}}
+```
+task snapshot max_count
+512
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Max_Total_Count Command 
+The `max_total_count` command returns the maximum number of snapshot the system can sustain.
```
Comment (DjP-iX) [inline content/SCALECLIReference/Task/CLISnapshotTask.md:230]:
```suggestion
The `max_total_count` command returns the maximum number of snapshots the system can sustain.
```

File: content/SCALECLIReference/Task/CLISnapshotTask.md [hunk 2]
```diff
@@ -6,12 +6,370 @@ aliases:
...
+Use the <code>[query](#query-command)</code> command to find the list of snapshot tasks and the task ID. 
+Use the <code>[get_instance](#get-instance-command)</code> command to list the properties of the task ID before and after making changes to verify the task configuration.
+
+{{< include file="/_includes/CLI/CLICommandWIP.md" type="page" >}}
+{{< expand "Using the Update Command" "v" >}}
+The `update` has one required property, `id`. 
+Enter any of the property arguments found in the **[Create Command Properties](#create-command)** table to change the task settings.
+Enter the property argument for `id` using `=` to separate the property and value. 
+Enter property arguments with string values with the value double-quoted.
+Enclose property arguments with array values in curly brackets `{}`, then enclose each propery argument in square brackets `[]` and sparate each with a comma and space.
```
Comment (DjP-iX) [inline content/SCALECLIReference/Task/CLISnapshotTask.md:320]:
```suggestion
Enclose property arguments with array values in curly brackets `{}`, then enclose each property argument in square brackets `[]` and separate each with a comma and space. 
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]

File: content/SCALECLIReference/Task/CLISnapshotTask.md [hunk 3]
```diff
@@ -6,12 +6,370 @@ aliases:
 draft: false
 tags:
 - scaleclitask
-- scalesnapshot
+- scalesnapshots
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALECLIReference/Task/CLISnapshotTask.md:9]: sound good.

File: content/SCALECLIReference/Task/CLISnapshotTask.md [hunk 4]
```diff
@@ -6,12 +6,370 @@ aliases:
...
+The following **snapshot** commands allow you to create new and manage existing periodic snapshot tasks.
+
+You can enter commands from the main CLI prompt or from the task snapshot namespace prompt.
+
+### Interactive Argument Editor (TUI)
+
+{{< include file="/_includes/CLI/HintInteractiveArgsEditor.md" type="page" >}}
+
+### Create Command 
+The `create` command creates a task to take snapshots of a given dataset at the specified schedule.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALECLIReference/Task/CLISnapshotTask.md:30]: I was being lazy here and economizing on words. I'll fix this where this occurs.

File: content/SCALECLIReference/Task/CLISnapshotTask.md [hunk 5]
```diff
@@ -6,12 +6,370 @@ aliases:
...
...accept default values for schedule properties, or enter each property argument enclosed in square brackets with double-quoted properties and values. Separate each array property argument enclosed in square brackets `[]` with a comma and space. Properties are: <br><li>`minute` specified in the format of minutes:seconds, or use the default 00:00 <br><li>`hour` specified in the format of 00 (0-23), or use the default `*` for every hour. <br><li>`dom` specify the day of month in the format of `Jan` through `Dec`, or use the default `*` for every month. <br><li>`month` specify the month specified as `jan` or use the default `*` for any month. <br><li>`dow` specify the day of week as `sun`, `mon`, `tue`, `wed`, `thu`, `fri`, or `sat`, or use the default `*` for every day of the week. <br><li>`begin` specify the earliest time the task starts in hour:minute or use the default `00:00`. <br><li>`end` specify the latest time the task can end in hour:minute, or use the default `00:00`.</li> Command example shows the default values for each property in the array. | <code>schedule={["minute"="00"], ["hour"="*"], ["dom"="*"], ["month"="*"], ["dow"="*"],["begin"="00:00"], ["end"="00:00"]}</code> |
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALECLIReference/Task/CLISnapshotTask.md]: fixed issues. Removed the commas, reworded the default value sentence, dom and month are very similar so the descriptions are correct, and I fixed the syntax that was missing a space.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**